src/util-ioctl.c

```diff
@@ -124,11 +124,12 @@ int GetIfaceMTU(const char *pcap_dev)
  * for the link. In case of uncertainty, it will output a
  * majorant to be sure avoid the cost of dynamic allocation.
  *
- * \param Name of a network interface
+ * \param LiveDevice object
  * \retval 0 in case of error
  */
-int GetIfaceMaxPacketSize(const char *pcap_dev)
+int GetIfaceMaxPacketSize(LiveDevice *ld)
 {
+    const char *pcap_dev = ld->dev;
```

Comment:
as a minor unrelated cleanup, lets rename this to just `dev`. It's not specific to pcap.